### DISCUSSION TITLE
ScriptReference/JsonUtility

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#688] [Notes]
# ScriptReference/JsonUtility

https://docs.unity3d.com/ScriptReference/JsonUtility.html

**Comment by cmcpasserby** [comment]
Only supports the same types Unity supports with its own serialization system and in the inspector (no Dictionary). For a more complete json implementation can use https://docs.unity3d.com/Packages/com.unity.nuget.newtonsoft-json@3.2/manual/index.html instead.
